lib/wagn/application.rb

```diff
@@ -106,10 +109,10 @@ def paths
         add_gem_path paths, 'gem-mod',             :with => 'mod'
         add_gem_path paths, 'gem-assets',          :with => 'public/assets'
 
-        paths['app/models'] = []
-        paths['app/mailers'] = []
-        paths['app/views'] = File.join( Wagn.gem_root, 'lib/card' )
-        paths['local-mod'] = approot_is_gemroot? ? [] : 'mod'
+#        paths['app/models'] = []
+#        paths['app/mailers'] = []
+#        paths['app/views'] = File.join( Wagn.gem_root, 'lib/card' )
```

Comment:
you can't do []= on paths any more.

They actually restructured the path handling considerably.